### PR TITLE
Adding description to plugin

### PR DESCRIPTION
**Pull request by kiiadi**

### CODE REVIEW

File: jetbrains-core/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -7,7 +7,19 @@
     <version>1.0</version>
 
     <description><![CDATA[
-      Plugin for interacting with AWS resources.
+<p>The AWS Toolkit is an open-source plug-in for JetBrains IDEs that makes it easier to develop applications built on AWS. The toolkit helps you create test, and debug serverless applications built using the AWS Serverless Application Model (<a href="https://aws.amazon.com/serverless/sam/">SAM</a>).
```
Comment (abrooksv) [inline jetbrains-core/resources/META-INF/plugin.xml]: Comma after create.

File: jetbrains-core/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -7,7 +7,19 @@
...
+</p>
+
+<p><h2><strong>Features</strong></h2></p>
+
+<ul>
+    <li><strong>New Project Wizard</strong> - Get started quickly by using one of the quickstart serverless application templates</li>
+    <li><strong>Run/Debug Local Lambda Functions</strong> - Locally test and step-through debug functions in a Lambda-like execution environment provided by the <a href="https://github.com/awslabs/aws-sam-cli">AWS SAM CLI</a></li>
```
Comment (abrooksv) [inline jetbrains-core/resources/META-INF/plugin.xml]: Says python Java only?

File: jetbrains-core/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -7,7 +7,19 @@
...
+    <li><strong>Resource Explorer</strong> - View your AWS Lambda remote functions & related CloudFormation stacks</li>
+    <li><strong>Invoke Remote Lambda Functions</strong> - Invoke remote functions using a sharable run-configuration</li>
+    <li><strong>Deploy SAM-based Application</strong> - (<strong>Python only</strong>) Package, deploy & track SAM-based applications</li>
```
Comment (zhangzhx) [inline jetbrains-core/resources/META-INF/plugin.xml]: Deploy SAM-based Application[s]? Plural?

File: jetbrains-core/resources/META-INF/plugin.xml (hunk 2 above)
Comment (kiiadi) [reply jetbrains-core/resources/META-INF/plugin.xml]: I've manually updated it in the plugin repo so it's only available in pycharm & intellij

**Comment by kiiadi** [comment]
@zhangzhx @abrooksv fixed your comments

**Pull request merged**